client-js/app/scripts/app.ts

```diff
@@ -141,13 +141,20 @@ window.addEventListener('WebComponentsReady', () => {
 
   class RegistryActivity implements Activity {
     path: Array<string>;
+    private globElem: LabradRegistry;
 
-    constructor(path: Array<string>) {
+    private api: registry.RegistryServiceJsonRpc;
```

Comment:
Use the interface type here (`registry.RegistryApi`) rather than an implementation type. This allows swapping out implementations, e.g. for tests.